tests/override_config_table/test_override_config_table_masic.py

```diff
@@ -48,6 +48,8 @@ def setup_env(duthost, tbinfo):
         original_pfcwd_value = update_pfcwd_default_state(duthost, "/etc/sonic/init_cfg.json", "disable")
     # Backup configDB
     for asic_id in duthost.get_asic_ids():
+        if asic_id is None:
+            continue
```

Comment:
The for loop is to backp and restore config.
```
cp etc/sonic/config_db.json /etc/sonic/config_db.json_before_override
cp etc/sonic/config_db0.json /etc/sonic/config_db0.json_before_override
...
cp etc/sonic/config_dbN.json /etc/sonic/config_dbN.json_before_override
```
Suggest change asic_id to "" when asic_id is None

Reply:
unfortunately, I tried your solution. It also failed during restore stage. as below:

```
    for asic_id in duthost.get_asic_ids():
        if asic_id is None:
            continue
        config = "/etc/sonic/config_db{}.json".format(asic_id)
        config_backup = "/etc/sonic/config_db{}.json_before_override".format(asic_id)
        restore_config(duthost, config, config_backup)   //  if convert asic_id to empty string, then this line is duplicated behavior with below line
    restore_config(duthost, CONFIG_DB, CONFIG_DB_BACKUP)  // will encounter command failure, because of above line had resorted config files.
```